spec/inferno/dsl/fhir_client_spec.rb

```diff
@@ -106,6 +179,31 @@ def test_session_id
       expect(group.request).to eq(result)
     end
 
+    context 'with a body of parameters' do
+      it 'uses get when all parameters are primitive' do
+        body = FHIR::Parameters.new
+        body.parameter = [primitive_param_string, primitive_param_bool]
+
+        query_string = '?prim_param_bool=true&prim_param_string=xyz'
+
+        get_with_body_request_stub =
+          stub_request(:get, "#{base_url}/#{path + query_string}")
```

Comment:
At some point, matching against literal query strings will drive you insane, so you should [use a hash instead](https://github.com/bblimke/webmock#matching-query-params-using-hash):
```ruby
stub_request(:get, "#{base_url}/#{path}")
  .with(query: { prim_param_string: 'xyz', ... })
  .to_return(...)
```